spec/hashie/mash_spec.rb

```diff
@@ -134,6 +134,19 @@
     expect(subject.type).to eq 'Steve'
   end
 
+  it 'logs a warning when overriding built-in methods' do
+    original_logger = Hashie.logger
+    stdout = StringIO.new
+    Hashie.logger = Logger.new(stdout)
```

Comment:
Should be around the test, `before` and `after`, maybe with `context`, otherwise on failure it doesn't reset.